devices/stm32l0x3.yaml

```diff
@@ -1,18 +1,115 @@
 _svd: ../svd/stm32l0x3.svd
 
-# Use LPUART1 cannonical name
 _modify:
+  # Use LPUART1 cannonical name
   LPUSART1:
     name: LPUART1
+  DBGMCU:
+    name: DBG
+  Flash:
+    name: FLASH
+  Firewall:
+    name: FW
+  SYSCFG_COMP:
+    name: SYSCFG
 
+_delete:
+  - USB_SRAM
+
+ADC:
+  CFGR1:
+    _modify:
+      AUTDLY:
+        name: WAIT
+  SMPR:
+    _modify:
+      SMPR:
+        name: SMP
+  CCR:
+    # VLCDEN is only available on l0x1 and l0x3
+    # It's also missing in the l0x3 SVD
+    _add:
+      VLCDEN:
+        bitOffset: 24
+        bitWidth: 1
+        description: "VLCD reading circuitry enable"
+        Disabled: [0, "VLCD reading circuitry disabled"]
+        Enabled: [1, "VLCD reading circuitry enabled"]
```

Comment:
This won't work to add the field values, you need to write them separately outside of the `_add` bit. It would suffice to just add `VLCDEN:` indented to the same level as `_add` after the `description` line.